src/ngraph/pass/assign_placement.cpp

```diff
@@ -26,8 +28,31 @@ pass::AssignPlacement::AssignPlacement(function<Placement(shared_ptr<Node>)> pla
 {
 }
 
+pass::AssignPlacement::AssignPlacement(vector<shared_ptr<runtime::Backend>> placement_backends)
+    : m_placement_backends(placement_backends)
+{
+}
+
 bool pass::AssignPlacement::run_on_node(shared_ptr<Node> node)
 {
-    node->set_placement(m_placement_policy(node));
+    if (!m_placement_backends.empty())
+    {
+        size_t backend_indx = 0;
+        for (auto backend : m_placement_backends)
```

Comment:
nitpick: you could just use
```cpp
for (size_t backend_indx = 0; backend_indx < m_placement_backends.size(); backend_indx ++)
{
if (m_placement_backends.at(backend_indx) ->is_supported(*node))
            {
                node->set_placement(backend_indx);
                 return false;
            }
}

Reply:
nitpick:

why not ind **e** x? it's so easy to misspell `backend_indx`
`backend_indx`  -> `backend_index`